Tests/RoslynPluginGeneratorTests/RoslynPluginJarBuilderTests.cs

```diff
@@ -71,9 +69,11 @@ public void RoslynPlugin_Test()
                 "META-INF\\MANIFEST.MF",
                 "static\\foo.zip",
                 "org\\sonar\\plugins\\roslynsdk\\configuration.xml",
-                "org\\sonar\\plugins\\roslynsdk\\sqale.xml",
                 "org\\sonar\\plugins\\roslynsdk\\rules.xml"
                 );
+
+            checker.AssertZipDoesNotContainFiles(
```

Comment:
FYI this check currently fails because the template jar file does still contain a _sqale.xml_ file.
I've created a separate PR on the template repo to remove the file, and when that's committed I'll update the jar resource used by this project. The test should then pass.